scripts/zones/Horlais_Peak/mobs/Evil_Oscar.lua

```diff
@@ -26,18 +26,29 @@ local ID = require("scripts/zones/Horlais_Peak/IDs")
 -----------------------------------
 local entity = {}
 
-local sendMessage = function(players)
+local sendFillingMessage = function(players)
     for _, member in pairs(players) do
         member:messageSpecial(ID.text.EVIL_OSCAR_BEGINS_FILLING)
     end
 end
 
+local sendOutOfRangeMessage = function(players, target)
+    for _, member in pairs(players) do
+        member:messageBasic(4, 0, 0, target)
+    end
+end
+
 local evilOscarFillsHisLungs
 evilOscarFillsHisLungs = function(mob)
```

Comment:
This function declaration looks strange to me? Could be I've never seen it formatted this way. I know its not your work, just wasn't sure why its written this way.

Reply:
oh true. Maybe it's old styling that was never updated? lol

Reply:
> sure

Yeah true, does look weird, I can fix it

Reply:
yeah declaring the local value like that is due to the function being a recursive function, I changed it to use the lua syntactic sugar style (see https://stackoverflow.com/questions/48209461/global-and-local-recursive-functions-in-lua).